packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/modules/internals/DevMenuInternalWebBrowserModule.kt

```diff
@@ -0,0 +1,33 @@
+package expo.modules.devmenu.modules.internals
+
+import android.content.Intent
+import android.net.Uri
+import androidx.browser.customtabs.CustomTabsIntent
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReactApplicationContext
+import expo.modules.devmenu.modules.DevMenuInternalWebBrowserModuleInterface
+
+class DevMenuInternalWebBrowserModule(private val reactContext: ReactApplicationContext)
+  : DevMenuInternalWebBrowserModuleInterface {
+  override fun openWebBrowserAsync(startUrl: String?, promise: Promise) {
+    requireNotNull(startUrl)
+
+    val intent = createCustomTabsIntent()
+    intent.data = Uri.parse(startUrl)
+
+    reactContext.currentActivity!!.startActivity(intent)
```

Comment:
50/50 🤔 
shouldn't we handle the case when the activity is null and maybe reject?